docs/pages/get-started/expo-go.mdx

```diff
@@ -1,14 +1,19 @@
 ---
 title: Expo Go
-description: Learn about Expo Go, the free, open-source client for developing and testing React Native apps on Android and iOS.
+description: Learn about Expo Go, the free, open-source sandbox app for learning and experimenting with React Native on Android and iOS.
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { BookOpen02Icon } from '@expo/styleguide-icons';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
 
-[Expo Go](https://expo.dev/client) is a free, [open-source](https://github.com/expo/expo/tree/main/home) client for testing React Native apps on Android and iOS devices without building anything locally. It allows you to open up apps served through Expo CLI and run your projects faster when developing them.
+[Expo Go](https://expo.dev/client) is a free, [open-source](https://github.com/expo/expo/tree/main/home) sandbox for learning and experimenting with React Native on Android and iOS devices. You can install it directly from App Stores, and get up and running within minutes — no need to install a native toolchain and compile an app.
```

Comment:
```suggestion
[Expo Go](https://expo.dev/client) is a free, [open-source](https://github.com/expo/expo/tree/main/home) sandbox for learning and experimenting with React Native on Android and iOS devices. You can install it directly from App Stores, and get up and running within minutes &mdash; no need to install a native toolchain and compile an app.
```

To use a dash, let's use `&mdash;` otherwise, we can also split the sentence.